docker_config/dockerfiles/Dockerfile.worker

```diff
@@ -62,9 +62,9 @@ RUN mkdir ${WORKDIR}/scripts
 
 # Install dependencies
 COPY requirements.txt requirements.txt
-RUN python3.6 -m pip install --user --upgrade pip==20.3.4; \
-    python3.6 -m pip install setuptools --upgrade; \
-    python3.6 -m pip install --no-cache-dir -r requirements.txt;
+RUN python3 -m pip install --user --upgrade pip==20.3.4; \
+    python3 -m pip install setuptools --upgrade; \
+    python3 -m pip install --no-cache-dir -r requirements.txt;
```

Comment:
Just making sure that python3 ends up pointing to 3.7, rather than the default system installation of python (3.5?), right?

Reply:
yup, one of the lines above prepends the conda `bin` directory to $PATH, which should set the default python.